features/step_definitions/api-token.js

```diff
@@ -47,13 +48,15 @@ module.exports = function server() {
                 bearer: this.jwt
             }
         }).then(() => this.getJwt(this.testToken.value).then((response) => {
-            Assert.strictEqual(response.statusCode, 200);
+            Assert.oneOf(response.statusCode, [200, 401]);
```

Comment:
Basically what Tiff said.

This step gets used by both the `create` test, and the `delete` test. In the `create` test we're logging in with a valid token, so we expect the response to be 200 (which is checked in the [step for that case](https://github.com/screwdriver-cd/screwdriver/blob/user-tokens/features/step_definitions/api-token.js#L56) too).

In the `delete` test, we're calling login with an invalid token, so we expect the response to be `401`. Again in that case it's further checked in the [next step](https://github.com/screwdriver-cd/screwdriver/blob/user-tokens/features/step_definitions/api-token.js#L183) to make sure we got the right one of the two.